streaming/java/streaming-api/src/main/java/org/ray/streaming/api/Language.java

```diff
@@ -0,0 +1,6 @@
+package org.ray.streaming.api;
+
+public enum Language {
```

Comment:
just use `org.ray.runtime.generated.Common.Language`?

Reply:
I thought that before. But that will make `ray-runtime` as a dependency of `streaming-api` . I think `streaming-api` should only depends on `ray-api` jar. And Language supported in streaming is more limited than ray core,. For example `CPP` won't be supported in streaming.

Reply:
use `org.ray.runtime.generated.Common.Language` now